contexts/AutocratContext.tsx

```diff
@@ -75,12 +67,12 @@ export function AutocratProvider({ children }: { children: ReactNode }) {
     }
     return new Program<OpenbookV2>(OPENBOOK_IDL, OPENBOOK_PROGRAM_ID, provider);
   }, [provider]);
+
   const { program: openbookTwap } = useOpenbookTwap();
   const { program: vaultProgram } = useConditionalVault();
   const [daoState, setDaoState] = useState<DaoState>();
   const [proposals, setProposals] = useState<ProposalAccountWithKey[]>();
-  const [allMarketsInfo, setAllMarketsInfo] = useState<AllMarketsInfo>({});
```

Comment:
My concern is that market info should stay in the AutocratContext otherwise they will be refetched every time a proposal context is created.

This can be experimented by testing this version against the current master and doing: proposal list -> trade one proposal -> go back to list -> trade another proposal. Current master uses 4 fetchs, while this branch does a dozen.

Reply:
I moved MarketCard to the proposal page, it solves this issue while being more clear on where things are : in proposal list there is a very brief overview, in proposal : an in depth view with both the MarketCard and the orderbooks. We could add more basic infos in the proposals array (ProposalAccountWithKey[]) if we want